### DISCUSSION TITLE
EPG for canalplus africa

### DISCUSSION THREAD

**Discussion by MagicOneFr** [iptv-org/community#962] [Q&A]
Hello,
It should be possible to grab the epg for canal plus africa from the same website used for "France metropolitaine".
In the current config file I can see some urls corresponding to african countries (I guess) but I've no idea how to use them.
Is it possible? If yes how?
Thank you.

**Comment by freearhey** [comment]
At the moment, using the canalplus.com config it is possible to download the guide for African countries as well.

https://github.com/iptv-org/epg/tree/master/sites/canalplus.com
